pkg/task/task.go

```diff
@@ -657,6 +661,34 @@ func (m *taskExecutor) patchOEV1alpha1SPC() (err error) {
 	return
 }
 
+// patchOEV1alpha1CSPC will patch a CSPC object in a kubernetes cluster.
+// The patch specifications as configured in the RunTask
+func (m *taskExecutor) patchOEV1alpha1CSPC() (err error) {
```

Comment:
For every new actions, we are making use of builder directly.
Please search for the word `builder` in this file to get references

Reply:
This can be done later if required !!!